app/views/homepage/index.html.erb

```diff
@@ -1,12 +1,8 @@
 <div id='homepage'>
-
-
-<p>
-
-
-<%= form_tag(games_path, :method => "get", id: "search-form") do %>
-   <%= text_field_tag :search, params[:search], placeholder: "Search Games" %>
-   <div class='text-center'><%= submit_tag "Search", :name => nil, id: 'black'%> </div>
-<% end %>
-</p>
+  <p>
+    <%= form_tag(games_path, :method => "get", id: "search-form") do %>
+       <%= text_field_tag :search, params[:search], placeholder: "Search Games" %>
+       <div class='text-center'><%= submit_tag "Search", :name => nil%> </div>
+    <% end %>
+  </p>
 </div>
```

Comment:
I personally detest the hashrocket syntax, and strive to avoid it wherever possible. So I would refactor this as, e.g. `form_tag(games_path, method: :get, id: "search-form") do`, and so forth.